### PR TITLE
[CCCP-415] Move psbt_signer into periodic

### PR DESCRIPTION
**Pull request by alstjd0921**
## Description

* `PsbtSigner` moved into periodic worker

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Something else (simple changes that are not related to existing functionality or others)

# Checklist

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have made new test codes regarding to my changes.
- [ ] I have no personal secrets or credentials described on my changes.
- [ ] I have run `cargo-clippy`  and linted my code.
- [ ] My changes generate no new warnings.
- [ ] My changes passed the existing test codes.
- [ ] My changes are able to compile.

### CODE REVIEW

File: primitives/src/constants/schedule.rs [hunk 1]
```diff
@@ -24,3 +24,6 @@ pub const MIGRATION_DETECTOR_SCHEDULE: &str = "*/1 * * * * * *";
 
 /// The schedule definition for presubmission. This will trigger on every hour.
 pub const PRESUBMISSION_SCHEDULE: &str = "0 0 * * * * *";
+
+/// The schedule definition for PSBT signer. This will trigger on every 3rd second.
+pub const PSBT_SIGNER_SCHEDULE: &str = "*/3 * * * * * *";
```
Comment (dnjscksdn98) [inline primitives/src/constants/schedule.rs:29]: 매 3초마다 키스토어 접근해서 서명 시도를 할텐데, 괜찮을까요?

Comment (dnjscksdn98) [reply primitives/src/constants/schedule.rs:29]: https://github.com/bifrost-platform/bifrost-relayer.rs/blob/pre-v2.0.0/client/src/btc/handlers/psbt_signer.rs#L118

지금은 서명된 값이 중복적용이 안되게끔만 막혀져 있어요

File: periodic/src/psbt_signer.rs [hunk 2]
```diff
@@ -31,36 +26,32 @@ pub struct PsbtSigner<T> {
...
-		event_receiver: Receiver<BTCEventMessage>,
 		keypair_storage: Arc<RwLock<KeypairStorage>>,
 		migration_sequence: Arc<RwLock<MigrationSequence>>,
 		btc_network: Network,
 	) -> Self {
 		Self {
 			client,
 			xt_request_sender,
-			event_receiver,
-			target_event: EventType::NewBlock,
```
[the remark this answers is not in the document]
Comment (dnjscksdn98) [reply periodic/src/psbt_signer.rs:60]: 필요없다면 지워도 될것 같아요

[the remark this answers is not in the document]
Comment (alstjd0921) [reply periodic/src/psbt_signer.rs:60]: 지워져있는겁니다

[the remark this answers is not in the document]
Comment (dnjscksdn98) [reply periodic/src/psbt_signer.rs:60]:
```
pub enum EventType {
	/// An inbound action.
	Inbound,
	/// An outbound action.
	Outbound,
	/// A new block mined.
	NewBlock,
}
```

아 여기서 말하는거였어요. enum 자체에서요

[the remark this answers is not in the document]
Comment (alstjd0921) [reply periodic/src/psbt_signer.rs:60]: 4a87d275

File: primitives/src/constants/schedule.rs (hunk 1 above)
Comment (alstjd0921) [reply primitives/src/constants/schedule.rs:29]: df0eee72

**Review by dnjscksdn98 — APPROVED** [review]

**Pull request merged**